ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -271,8 +271,8 @@ export default {
   timelineFile: 'e12s.txt',
   triggers: [
     {
-      // Headmarkers are randomized, so handle them all with a single trigger.
-      id: 'E12S Promise Headmarker',
+      // Headmarkers are randomized, so use a generic headMarker regex with no criteria.
+      id: 'E12S Promise Headmarker - Formless Judgement',
```

Comment:
```suggestion
      id: 'E12S Promise Formless Judgement',
```

Reply:
This would conflict with the trigger at line 459:

```js
    {
      id: 'E12S Promise Formless Judgment',
      netRegex: NetRegexes.startsUsing({ source: 'Eden\'s Promise', id: '58A9', capture: false }),
```

Reply:
`E12S Promise Formless Judgment You` then?